client/lib/plans/features-list.js

```diff
@@ -1315,6 +1319,26 @@ export const FEATURES_LIST = {
 		getTitle: () => i18n.translate( 'Live chat support 24X7' ),
 		getDescription: () => i18n.translate( 'Live chat is available 24/7.' ),
 	},
+
+	[ FEATURE_JETPACK_VIDEOPRESS ]: {
+		getSlug: () => FEATURE_JETPACK_VIDEOPRESS,
+		getTitle: () => i18n.translate( 'Unlimited users' ),
+	},
+
+	[ FEATURE_JETPACK_VIDEOPRESS_EDITOR ]: {
+		getSlug: () => FEATURE_JETPACK_VIDEOPRESS_EDITOR,
+		getTitle: () => i18n.translate( 'Built into WordPress editor' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **1** times:
`translate( 'Built into WordPress Editor' )` *ES Score: 10*